Classes/OldModel/Event.php

```diff
@@ -593,14 +560,29 @@ protected function getPlaceWithDetailsRaw(): string
      */
     protected function getPlacesAsArray(): array
     {
-        return \Tx_Oelib_Db::selectMultiple(
-            'title, address, zip, city, country, homepage, directions',
-            'tx_seminars_sites, tx_seminars_seminars_place_mm',
-            'uid_local = ' . $this->getUid() . ' AND uid = uid_foreign' .
-            \Tx_Oelib_Db::enableFields('tx_seminars_sites'),
-            '',
-            'sorting ASC'
-        );
+        $queryBuilder = self::getQueryBuilderForTable('tx_seminars_sites');
```

Comment:
`getPlacesAsArray` should use `getAssociationAsArray`, too.

Reply:
There is a difference in the `where` part of the two queries. `getAssociationAsArray()` uses `$this->getTopicOrSelfUid()` while `getPlacesAsArray()` uses `$this->getUid()`.